acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -206,20 +214,28 @@ export default async function init(element) {
   widgetContainer.className = `fsw wapper-${VERB}`;
   widget.appendChild(widgetContainer);
 
-  const isReturningUser = window.localStorage.getItem('pdfnow.auth');
   const isRedirection = /redirect_(?:conversion|files)=true/.test(window.location.search);
-  const preRenderDropZone = !isReturningUser && !isRedirection;
+  let isLimitExhausted = false;
+  const { cookie: cookies } = document;
+  const isExportVerb = VERB.startsWith('pdf-to');
+  const isCreateVerb = VERB.endsWith('to-pdf') || VERB === 'createpdf';
+  if (exhLimitCookieMap[VERB]) {
+    isLimitExhausted = cookies.includes(exhLimitCookieMap[VERB]);
+  } else if (isExportVerb) {
+    isLimitExhausted = cookies.includes(exhLimitCookieMap['pdf-to']);
+  } else if (isCreateVerb) {
+    isLimitExhausted = cookies.includes(exhLimitCookieMap['to-pdf']);
+  }
```

Comment:
`creatpdf` is an odd ball. If moving it to `exhLimitCookieMap`
```
const exhLimitCookieMap = {
  'to-pdf': 'p_ac_cr_p_c',
  'pdf-to': 'p_ac_ex_p_c',
  'compress-pdf': 'p_ac_cm_p_ops',
  'rotate-pages': 'p_ac_or_p_c',
  'createpdf': 'p_ac_cr_p_c',
};
```
The code can be simplified a lot.
```
const limitCookie = exhLimitCookieMap[VERB] || exhLimitCookieMap[VERB.match(/^pdf-to|to-pdf$/)?.[0]]
const isLimitExhausted = cookies.includes(limitCookie);
```

Reply:
Good point, updated.